### PR TITLE
[Design updates] -  confirm which programme first time through service

### PR DESCRIPTION
**Pull request by etheya**
Screens to follow. 
Includes, correct styling, headers, changed content.

### CODE REVIEW

File: app/views/privacy_policies/show.html.erb [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-three-quarters">
-    <h1 class="govuk-heading-xl">Privacy policy</h1>
+    <div class="govuk-grid-column-two-thirds">
 
-    <% if !@policy.acceptance_required?(current_user) %>
-      <% content_for :before_content, govuk_back_link(text: "Back", href: :back) %>
-    <% else %>
-      <p>Before you continue, please check our privacy policy</p>
-      <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
```
Comment (jamesrose) [inline app/views/privacy_policies/show.html.erb:11]: Are we ok removing this condition?

File: db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_04_29_144309) do
```
Comment (jamesrose) [inline db/schema.rb:13]: Can you run `rake db:migrate` locally and commit the changes? I don't think this change is relevant to the PR.

File: app/views/privacy_policies/show.html.erb (hunk 1 above)
Comment (etheya) [reply app/views/privacy_policies/show.html.erb:11]: Well the condition is still there other than the line ( Before you continue and the <hr> ) the rest should still be there. 

Not sure what the line of content saying that does for a user to be fair. Condition for acceptance is still in place isnt it @jamesrose

File: db/schema.rb (hunk 2 above)
Comment (etheya) [reply db/schema.rb:13]: No it shouldnt be.. i did do it - not sure why its pushing up that file. Can we not remove it from the PR @jamesrose

**Review by jamesrose — APPROVED** [review]

**Comment by jamesrose** [comment]
@etheya I fixed this one for you. Merge when you're ready.

**Comment by etheya** [comment]
Done - thanks

**Pull request merged**